app/views/split_checkout/_form.html.haml

```diff
@@ -2,5 +2,7 @@
   = inject_saved_credit_cards
 
 %div.checkout-step.medium-6
-  = form_with url: checkout_update_path(checkout_step), model: @order, method: :put do |f|
-    = render "split_checkout/#{checkout_step}", f: f
+  = form_with url: checkout_update_path(checkout_step), model: @order, method: :put,
+    data: { remote: "true" } do |form|
```

Comment:
Adding `data-remote: true` is the only real change needed on the frontend to convert the whole form submission to Ajax :ok_hand: